.github/workflows/integration_test.yaml

```diff
@@ -5,7 +5,7 @@ on:
 
 jobs:
   integration-tests:
-    uses: canonical/operator-workflows/.github/workflows/integration_test.yaml@main
+    uses: canonical/operator-workflows/.github/workflows/integration_test.yaml@debug-2
```

Comment:
I assume we're not closing this until this points back at `main`?

Reply:
Yes